src/api/users.js

```diff
@@ -11,28 +11,30 @@ const users_container = {
       access_token: '',
       user_details: {},
       user_webapps: [],
-      user_groups: {}
+      user_groups: []
     },
 
     getters: {
       get_user_details (state) { return state.user_details },
       get_user_id (state) { return state.user_id },
       get_user_firstname (state) { return state.user_firstname },
       get_user_webapps (state) { return state.user_webapps },
-      get_access_token (state) { return state.access_token }
+      get_access_token (state) { return state.access_token },
+      get_user_groups(state) { return state.user_groups }
     },
 
     mutations: {
       set_user_id (state, id) { state.user_id = id },
       set_user_firstname (state, name) { state.user_firstname = name },
       set_user_details (state, user_details) { state.user_details = user_details },
       set_user_webapps (state, webapps) { state.user_webapps = webapps },
-      set_access_token (state, token) { state.access_token = token }
+      set_access_token (state, token) { state.access_token = token },
+      set_user_groups (state, user_groups) { state.user_groups = user_groups }
     },
 
     actions: {
-      async get_user_details (context, field) {
-        const response = await backend.readByField('/users', field)
+      async get_user_details (context, id) {
+        const response = await backend.readByField('/users', id)
```

Comment:
Just a personal opinion:

Given the fact that "/users" is not a valid endpoint in itself, I find this kind of code line rather hard to read. Either provide fully valid relative URLs to your method, or hide the URL endpoint completely in the backen method.

Reply:
I changed it